cumulusci/cli/cci.py

```diff
@@ -136,34 +137,41 @@ def show_debug_info():
     pdb.post_mortem()
 
 
-@click.group("main", help="")
-def cli():
-    """Top-level `click` command group."""
-
-
-@cli.command(name="version", help="Print the current version of CumulusCI")
-def version():
-    click.echo("CumulusCI version: ", nl=False)
-    click.echo(click.style(cumulusci.__version__, bold=True), nl=False)
-    click.echo(f" ({sys.argv[0]})")
-    click.echo(f"Python version: {sys.version.split()[0]}", nl=False)
-    click.echo(f" ({sys.executable})")
+def show_version_info():
+    console = rich.get_console()
+    console.print(
+        f"[blue bold]CumulusCI version: [blink] {cumulusci.__version__}[/] ({sys.argv[0]})"
+    )
+    console.print(
+        f"[purple bold]Python version: {sys.version.split()[0]} ({sys.executable})"
+    )
+    console.print()
 
-    click.echo()
     current_version = get_installed_version()
     latest_version = get_latest_final_version()
     if latest_version > current_version:
-        click.echo(
-            f"There is a newer version of CumulusCI available ({str(latest_version)})."
+        console.print(
+            f"[yellow flashing]There is a newer version of CumulusCI available ({str(latest_version)})."
```

Comment:
I don't see a test for this line of code.

Reply:
We don't have an explicit assert for this string, but we do have coverage of this if block by mocking `get_latest_final_version` in the [test_version](https://github.com/SFDO-Tooling/CumulusCI/blob/344c6913a70d0a7d71ecced047653c0d7383e974/cumulusci/cli/tests/test_cci.py#L399) method.

Reply:
The `test_cci.py::test_version()` method does cover this if block, but probably a good call to add an assert for at least one string in this block.